admin/acceptance_tests/role_test.py

```diff
@@ -223,3 +226,33 @@ def test_grid_dberror(self, dbsession):
         )
         info = RoleViews(request).grid()
         assert info.status_int == 500, 'Expected 500 status when db error'
+
+
+@skip_if_ci
+@pytest.mark.selenium
+@pytest.mark.usefixtures('selenium', 'selenium_app', 'roles_test_data')
+class TestRoleSelenium():
+
+    _prefix = '/roles'
+
+    def test_index(self, selenium, selenium_app, roles_test_data, dbsession):
+        selenium.get(selenium_app + self._prefix)
+
+        index_page = IndexPage(selenium)
+        index_page.select_language('en')
+        index_page.check_pagination_info('Showing 1 to 23 of 23 rows', 10)
+        index_page.select_page_size(10)
+        index_page.check_pagination_info('Showing 1 to 10 of 23 rows', 10)
+
+        # delete
+        role = roles_test_data['roles'][3]
+        index_page.click_delete(role.id)
+        index_page.check_pagination_info('Showing 1 to 10 of 22 rows', 10)
+        with pytest.raises(NoSuchElementException):
+            selenium.find_element_by_xpath("//a[@data-url='{}/roles/{}')]"
```

Comment:
now that delete button is in a dropdown, it should not be accessible before click on dropdown, so this is always true.

Reply:
I do let a comment in place, could be linked with "One click edit from grid (layer tree ?)" issue.
Initially, the purpose was to ensure that the data / the page are reloaded.